### ISSUE TITLE
Scene description and leg movement strings should be updated to be more clear

### ISSUE THREAD

**Issue by jhung** [phetsims/john-travoltage#117]
Based on user experiences, the descriptions of leg movement should be changed to match the visuals and the actual model. Instead of describing the foot position as "on / off" the carpet, it is rephrased now as whether it is rubbing the rug.

For a table summarizing the new strings, see [Scene Description and Leg Position Changes (Sep 2016)](https://wiki.fluidproject.org/display/fluid/PhET+John+Travoltage+Simulation+Design#PhETJohnTravoltageSimulationDesign-SceneDescriptionandLegPositionChanges%28Sep2016%29)

**Comment by jessegreenberg** [comment]
Changes for this issue were merged into master via https://github.com/phetsims/john-travoltage/pull/120.  Closing.
